spec/integration/image-streamer/api300/plan_scripts/update_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+klass = OneviewSDK::ImageStreamer::API300::PlanScripts
+RSpec.describe klass, integration_i3s: true, type: UPDATE do
+  include_context 'integration i3s api300 context'
+
+  describe '#update' do
+    it 'updates the name of the plan script' do
+      item = klass.find_by($client_i3s_300, name: PLAN_SCRIPT1_NAME).first
+      expect(item['uri']).not_to be_empty
```

Comment:
It's a bit weird expecting the URI to not be empty. If it exists it will never be empty, if doesn't exists it would be nil, and I really don't know if `NilClass#empty?` would return true.